webpack/assets/javascripts/react_app/components/passwordStrength/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactPasswordStrength from 'react-password-strength';
+import './password_strength.scss';
+
+class PasswordStrength extends React.Component {
+
+  render() {
+    const {className, id, name, error} = this.props.data;
+
+    return (
+      <div className={`form-group ${error.length > 0 ? 'has-error' : ''}`}>
+        <label className="col-md-2 control-label"> { __('Password') } </label>
```

Comment:
This label should include a `for` attribute to tie it to the input below for accessibility.